src/main/java/org/opentripplanner/transit/service/StopModel.java

```diff
@@ -236,16 +236,22 @@ public void addTransitStopVertex(FeedScopedId id, TransitStopVertex stopVertex)
    */
   public void calculateTransitCenter() {
     var vertices = getAllStopVertices();
-    var medianCalculator = new MedianCalcForDoubles(vertices.size());
 
-    vertices.forEach(v -> medianCalculator.add(v.getLon()));
-    double lon = medianCalculator.median();
+    // we need this check because there could be only FlexStopLocations (which don't have vertices)
+    // in the graph
+    if (!vertices.isEmpty()) {
+      this.center = vertices.iterator().next().getCoordinate();
```

Comment:
```suggestion
```

Reply:
Thanks for spotting this. That must have been left over from an experiment.